lnd_test.go

```diff
@@ -43,7 +43,11 @@ var (
 )
 
 const (
-	testFeeBase = 1e+6
+	testFeeBase         = 1e+6
+	defaultTimeout      = time.Second * 30
```

Comment:
It'd be helpful to define these in `lntest`, and export them so that we can use them in other places other than the main test package. Could then write this section as:
```golang
    defaultTimeout = lntest.DefaultTimeout
    ....
```